dowhy/causal_discovery.py

```diff
@@ -0,0 +1,32 @@
+class GraphLearner:
```

Comment:
can you name the file "graph_learner.py" to be consistent with graphlearner class?